onyo/main.py

```diff
@@ -216,20 +240,21 @@ def setup_parser() -> OnyoArgumentParser:
     r"""Setup and return a fully populated OnyoArgumentParser for Onyo and all subcommands.
     """
     from onyo.onyo_arguments import args_onyo
-    from onyo.cli.cat import args_cat
-    from onyo.cli.config import args_config
-    from onyo.cli.edit import args_edit
-    from onyo.cli.get import args_get
-    from onyo.cli.history import args_history
-    from onyo.cli.init import args_init
-    from onyo.cli.mkdir import args_mkdir
-    from onyo.cli.mv import args_mv
+    from onyo.cli.cat import args_cat, epilog_cat
+    from onyo.cli.config import args_config, epilog_config
+    from onyo.cli.edit import args_edit, epilog_edit
+    from onyo.cli.fsck import epilog_fsck
+    from onyo.cli.get import args_get, epilog_get
+    from onyo.cli.history import args_history, epilog_history
+    from onyo.cli.init import args_init, epilog_init
+    from onyo.cli.mkdir import args_mkdir, epilog_mkdir
+    from onyo.cli.mv import args_mv, epilog_mv
     from onyo.cli.new import args_new
```

Comment:
I did not investigate, but `new.py` does also contain an epilog, even though it is here the only line the diff states wasn't changed to include the `epilog_new`.

If there was a reason for this, feel free to ignore and resolve.

Reply:
Good catch. I somehow missed staging this.

I have fixed this (and also added another rendering fix as a commit that I forgot).

Reply:
I have fixed this and pushed.